fxa-client/src/config.rs

```diff
@@ -26,6 +26,16 @@ struct OpenIdConfigurationResponse {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Config {
     content_url: String,
```

Comment:
Leaving `content_url` is a good call, it might be useful one day if we want to do regular re-fetches of the `RemoteConfig`.